plugin.audio.ampache/addon.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.audio.ampache" version="1.2.0" name="Ampache" provider-name="lusum">
+    <requires>
+        <import addon="script.module.future" version="0.16.0"/>
+    </requires>
+  <extension point="xbmc.python.pluginsource" library="default.py">
+    <provides>audio</provides>
+  </extension>
+  <extension point="xbmc.service" library="resources/lib/service.py" start="login" />
+  <extension point="xbmc.python.module" library="resources/lib/"/>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Stream music from Ampache XML-API</summary>
+    <summary lang="de_DE">Streame Musik von der Ampache XML-API</summary>
+    <summary lang="it_IT">Streaming Musicale per l'XML-API di Ampache</summary>
+    <description lang="en_GB">A web based audio/video streaming application and file manager allowing you to access your music and videos from anywhere, using almost any internet enabled device.</description>
+    <platform>all</platform>
+    <forum>https://forum.kodi.tv/showthread.php?tid=230736</forum>
+    <website>http://ampache.org/</website>
```

Comment:
since you'll have to touch this file please reindent it properly

Reply:
Ok, i will indent

Reply:
Btw, i have checked other plugins and it is indented in the same way, maybe you can give me an example of a plugin with a correct indentation?

Reply:
Ok, found the problem

Reply:
done